webapp/publisher/views.py

```diff
@@ -19,9 +19,22 @@
 from webapp.solutions.logic import (
     get_publisher_solutions,
     publisher_has_solutions,
+    register_solution,
+    get_user_teams_for_solutions,
 )
 
 
+def _check_solutions_access(username):
+    """
+    Check if user has solutions access
+    Redirect to /charms if not
+    """
+    has_solutions = publisher_has_solutions(username)
+    if not has_solutions:
+        return redirect("/charms")
```

Comment:
Is there a way to let users know the reason why they're being redirected to charms? I think this might be a bit confusing for users, they might not know the reason they are being redirect to charms is because they do not have solutions.